gips/data/gpm/gpm.py

```diff
@@ -0,0 +1,335 @@
+                ftp.quit()
+                os.rename(temp_fp, stage_fp)
+            return [stage_fp]
+        return []
+
+class gpmData(Data):
+    """ A tile of data (all assets and products) """
+    name = 'GPM'
+    version = '1.0.0'
+    Asset = gpmAsset
+
+    _projection = 'GEOGCS["WGS 84",DATUM["WGS_1984",SPHEROID["WGS 84",6378137,298.2572326660159,' \
+                  'AUTHORITY["EPSG","7030"]], AUTHORITY["EPSG","6326"]],PRIMEM["Greenwich",0],' \
+                  'UNIT["degree",0.0174532925199433],AUTHORITY["EPSG","4326"]]'
+
+    _products = {
+        'pratefinal': {
+            'description': 'Final Precipitation Rate Averaged over 1 Day in mm/hr',
+            # the list of asset types associated with this product
+            'assets': ['IMERG-DAY-FINAL'],
+            'startdate': datetime.date(2014, 3, 12),
+            'sensor': 'GPM',
+            '_geotransform': (-179.9499969, 0.10000000149011612, 0.0, 89.9499969, 0.0, -0.10000000149011612),
+        },
+        'paccnrtgpmlate': {
+            'description': 'NRT Precipitation Accumulated over 1 Day ending at 23:59 UTC in mm - GPM (and friends) Archive',
+            # the list of asset types associated with this product
+            'assets': ['IMERG-DAY-LATE'],
+            'startdate': datetime.date(2014, 3, 12),
+            'sensor': 'GPM',
+            '_geotransform': (-179.9499969, 0.10000000149011612, 0.0, 89.9499969, 0.0, -0.10000000149011612),
+        },
+        'paccnrtgpmmid': {
+            'description': 'NRT Precipitation Accumulated over 1 Day ending at 14:59 UTC in mm - GPM (and friends) Archive',
+            # the list of asset types associated with this product
+            'assets': ['IMERG-DAY-MID'],
+            'startdate': datetime.date(2014, 3, 12),
+            'sensor': 'GPM',
+            '_geotransform': (-179.9499969, 0.10000000149011612, 0.0, 89.9499969, 0.0, -0.10000000149011612),
+        },
+        'paccnrtgpmearly': {
+            'description': 'NRT Precipitation Accumulated over 1 Day ending at 06:59 UTC in mm - GPM (and friends) Archive',
+            # the list of asset types associated with this product
+            'assets': ['IMERG-DAY-EARLY'],
+            'startdate': datetime.date(2014, 3, 12),
+            'sensor': 'GPM',
+            '_geotransform': (-179.9499969, 0.10000000149011612, 0.0, 89.9499969, 0.0, -0.10000000149011612),
+        },
+        'paccnrttrmm': {
+            'description': 'NRT Precipitation Accumulated over 1 Day in mm - TRMM (and friends) Archive',
+            # the list of asset types associated with this product
+            'assets': ['3B42-DAY-LATE'],
+            'startdate': datetime.date(2000, 3, 1),
+            'sensor': 'GPM',
+            '_geotransform': (-179.875, 0.25, 0.0, 59.875, 0.0, -0.25),
+        },
+    }
+
+    def asset_check(self, prod_type):
+        """Is an asset available for the current scene and product?
```

Comment:
How come you didn't pull the modis version up to the superclass and have them both drivers call in?  Now there's three times the code, counting the third copy in the smap driver.